test/cql-pytest/test_cast.py

```diff
@@ -47,7 +47,7 @@ def table2(cql, test_keyspace):
     cql.execute("DROP TABLE " + table)
 
 # Test casting int values to blob values using C-style casts
-def test_cast_int_blob(cql, table1):
+def test_cast_int_blob(cql, table1, scylla_only):
```

Comment:
In the first patch I suggested that you split this long test functions into several short test functions. Then you can mark individual tests scylla_only while the others remain true also on Cassandra. At a minimum you should split this into two tests - one scylla_only and the other one not - but I suggest splitting into more than two.

In general, as much as the test code as possible should not have scylla_only, otherwise it is impossible to verify that the test doesn't just enshrine a bug.